tokio-trace-env-logger/examples/hyper-echo.rs

```diff
@@ -49,7 +49,7 @@ fn echo(req: Request<Body>) -> Instrumented<BoxFut> {
                 let body = req.into_body();
                 *response.body_mut() = body;
                 (
-                    span!("response", response_kind = "echo"),
+                    span!("response", response_kind = &"echo"),
```

Comment:
I think I get _why_ this is the case, but it's kinda weird to see `&` on string literals.

Reply:
Yeah, I don't like it either, but I spent a long time struggling with this and _I_ couldn't find a way to make this API generic over borrowing and owning values...

Reply:
Maybe [this article](https://joshlf.com/post/2018/10/18/rust-higher-kinded-types-already/) could help until ATC's land?